lldb/test/API/lang/swift/foundation_value_types/measurement/main.swift

```diff
@@ -13,8 +13,8 @@ import Foundation
 
 func main() {
   var measurement = Measurement(value: 1.25, unit: Unit(symbol: "m"))
+  //% self.runCmd("settings set symbols.swift-enable-ast-context false")
```

Comment:
Can you add a comment explaining why this is here?

Ideally, we would convert this into a non-inline test and run the test with and without the setting, otherwise we're losing the typeref <-> SwiftAST validation.

Reply:
good call, done.